projects/core/src/occ/adapters/cart/converters/occ-cart-normalizer.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { TestBed } from '@angular/core/testing';
+import { PRODUCT_NORMALIZER } from '../../../../product/connectors/product/converters';
+import { ConverterService } from '../../../../util/converter.service';
+import { OccCartNormalizer } from './occ-cart-normalizer';
+
+describe('OccCartNormalizer', () => {
+  let service: OccCartNormalizer;
```

Comment:
Could we call the `service` variable something meaningful like `occCartNormalizer`.  The test logic is difficult to read using the `service` var name, especially since there is ConverterService in use as well.